Assets/Plugins/CountlySDK/Services/DeviceIdCountlyService.cs

```diff
@@ -15,30 +15,43 @@ public class DeviceIdCountlyService : AbstractBaseService
         private readonly EventCountlyService _eventCountlyService;
         internal readonly RequestCountlyHelper _requestCountlyHelper;
         private readonly SessionCountlyService _sessionCountlyService;
+        
 
         internal DeviceIdCountlyService(CountlyConfiguration configuration, CountlyLogHelper logHelper, SessionCountlyService sessionCountlyService,
             RequestCountlyHelper requestCountlyHelper, EventCountlyService eventCountlyService, CountlyUtils countlyUtils, ConsentCountlyService consentService) : base(configuration, logHelper, consentService)
         {
             Log.Debug("[DeviceIdCountlyService] Initializing.");
-
+            
             _countlyUtils = countlyUtils;
             _eventCountlyService = eventCountlyService;
             _requestCountlyHelper = requestCountlyHelper;
             _sessionCountlyService = sessionCountlyService;
+
+            int type = PlayerPrefs.GetInt(Constants.DeviceIDType, 0);
```

Comment:
device ID type acquisition should be together with setting the device ID as the type changes depending on the acquisition method.
The default ID type value should not be a valid value